scl/netskope/plugin.conf

```diff
@@ -0,0 +1,22 @@
+
+
+# sample log
+#   <134>{"count": 1, "supporting_data": {"data_values": ["x.x.x.x", "user@domain.com"], "data_type": "user"}, "organization_unit": "domain/domain/Domain Users/Enterprise Users", "severity_level": 2, "category": null, "timestamp": 1547421943, "_insertion_epoch_timestamp": 1547421943, "ccl": "unknown", "user": "user@domain.com", "audit_log_event": "Login Successful", "ur_normalized": "user@domain.com", "_id": "936289", "type": "admin_audit_logs", "appcategory": null}
+
+# assumed to be running with flags(no-parse), e.g. the entire incoming log
```

Comment:
Can we also assume that netskope logs are coming over network? In this case we can "include the `flags(no-parse)` assumption" by creating a whole source block instead of a parser.

Reply:
quoting a comment I sent via email: That is coming from default-network-drivers(). Theres no point in creating a separate source for all device types.